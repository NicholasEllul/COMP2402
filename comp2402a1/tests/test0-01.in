1. phil
2. am
3. iasdasdssad
4. hi