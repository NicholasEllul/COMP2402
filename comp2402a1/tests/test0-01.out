3. iasdasdssad
4. hi
2. am
1. phil
